randovania/data/json_data/prime1.json

```diff
@@ -16727,7 +16824,7 @@
                     "in_dark_aether": false,
                     "asset_id": 907887024,
                     "default_node_index": 1,
-                    "valid_starting_location": true,
+                    "valid_starting_location": false,
```

Comment:
This could be left as a valid start since it requires an event to leave anyway. Good for a potential room rando if that ever happens.

Reply:
I don't see how this one can ever be valid. Since the default spawn point is the save station, you'd be trapped in the little space on the wrong side of the gate

Reply:
Maybe not for the Save Station, unless the spawn is changed. But this is probably fine.